### ISSUE TITLE
App creation error

### ISSUE THREAD

**Issue by Super-Gil** [GenieFramework/Genie.jl#265]
**Describe the bug**
I can't create an app use Genie.newapp()

**Error stacktrace**
`Genie v0.31.5
Active env: DEV

Ready! 
ERROR: ArgParse.ArgParseError("too many arguments")
Stacktrace:
 [1] argparse_error(::Any) at C:\Users\Administrator\.julia\packages\ArgParse\WUIf8\src\parsing.jl:9
 [2] parse_arg!(::ArgParse.ParserState, ::ArgParse.ArgParseSettings) at C:\Users\Administrator\.julia\packages\ArgParse\WUIf8\src\parsing.jl:918
 [3] parse_args_unhandled(::Array{T,1} where T, ::ArgParse.ArgParseSettings, ::Bool) at C:\Users\Administrator\.julia\packages\ArgParse\WUIf8\src\parsing.jl:632
 [4] parse_args_unhandled at C:\Users\Administrator\.julia\packages\ArgParse\WUIf8\src\parsing.jl:595 [inlined]
 [5] parse_args(::Array{T,1} where T, ::ArgParse.ArgParseSettings; as_symbols::Bool) at C:\Users\Administrator\.julia\packages\ArgParse\WUIf8\src\parsing.jl:469
 [6] parse_args at C:\Users\Administrator\.julia\packages\ArgParse\WUIf8\src\parsing.jl:466 [inlined]
 [7] #parse_args#58 at C:\Users\Administrator\.julia\packages\ArgParse\WUIf8\src\parsing.jl:421 [inlined]
 [8] parse_args at C:\Users\Administrator\.julia\packages\ArgParse\WUIf8\src\parsing.jl:421 [inlined]
 [9] parse_commandline_args(::Genie.Configuration.Settings) at C:\Users\Administrator\.julia\packages\Genie\8CTam\src\Commands.jl:91
 [10] execute(::Genie.Configuration.Settings; server::Nothing) at C:\Users\Administrator\.julia\packages\Genie\8CTam\src\Commands.jl:17
 [11] #run#31 at C:\Users\Administrator\.julia\packages\Genie\8CTam\src\Genie.jl:238 [inlined]
 [12] top-level scope at A:\Course\julia\Urum\genie.jl:37
 [13] include(::Module, ::String) at .\Base.jl:377
 [14] include at A:\Course\julia\Urum\src\Urum.jl:1 [inlined]
 [15] main() at A:\Course\julia\Urum\src\Urum.jl:8
 [16] top-level scope at A:\Course\julia\Urum\src\Urum.jl:12
 [17] include(::String) at .\client.jl:439
 [18] main() at A:\Course\julia\Urum\bootstrap.jl:6
 [19] top-level scope at A:\Course\julia\Urum\bootstrap.jl:5
 [20] include(::String) at .\client.jl:439
 [21] top-level scope at C:\Users\Administrator\.julia\packages\Genie\8CTam\src\Genie.jl:187
 [22] eval at .\boot.jl:331 [inlined]
 [23] loadapp(::String; autostart::Bool) at C:\Users\Administrator\.julia\packages\Genie\8CTam\src\Genie.jl:186
 [24] autostart_app(::String; autostart::Bool) at C:\Users\Administrator\.julia\packages\Genie\8CTam\src\Generator.jl:322
 [25] newapp(::String; autostart::Bool, fullstack::Bool, dbsupport::Bool, mvcsupport::Bool, testmode::Bool) at C:\Users\Administrator\.julia\packages\Genie\8CTam\src\Generator.jl:429
 [26] newapp(::String) at C:\Users\Administrator\.julia\packages\Genie\8CTam\src\Generator.jl:396
 [27] top-level scope at REPL[7]:1`

**To reproduce**
Genie.newapp("Urum")

**Expected behavior**
Should work fine?

**Additional context**
Please include the output of 
`julia> versioninfo()`
and
`pkg> st`

### version info:
`julia> versioninfo()
Julia Version 1.4.2
Commit 44fa15b150* (2020-05-23 18:35 UTC)
Platform Info:
  OS: Windows (x86_64-w64-mingw32)
  CPU: Intel(R) Core(TM) i7-8750H CPU @ 2.20GHz
  WORD_SIZE: 64
  LIBM: libopenlibm
  LLVM: libLLVM-8.0.1 (ORCJIT, skylake)
Environment:
  JULIA_PKG_SERVER = https://cn-east.pkg.juliacn.com
  JULIA_REVISE = auto
  JULIA_EDITOR = "C:\Users\Administrator\AppData\Local\Programs\Microsoft VS Code\Code.exe"
  JULIA_NUM_THREADS =`

### result of st:
`(Urum) pkg> st
Status `A:\Course\julia\Urum\Project.toml`
  [c43c736e] Genie v0.31.5
  [e6f89c97] LoggingExtras v0.4.2
  [295af30f] Revise v2.7.3`

**Comment by essenciary** [comment]
@Super-Gil Thanks for reporting this - I tried everything but I can't reproduce. I tried various paths, all works perfectly - there's also a test suite checking app creation. 

Any other info you can provide so I can replicate? What terminal are you using (or how are you running this?).

**Comment by Super-Gil** [comment]
@essenciary Pretty much appreciated for your reply. I am using Windows10... And I think I did it in my VSCode terminal which performs the same as system cmd? (Not quite sure...) Getting pretty confused. Since I am a Julia newbie I might not be able to solve too advanced issues... 

I did remember I just did 
`using Pkg`
`Pkg.add("Genie")`
`Pkg.build("Genie")`
`using Genie`
`Genie.newapp("SomeProjects")`
